ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly-4.14-cpou-upgrade-from-4.12.yaml

```diff
@@ -59,6 +59,12 @@ releases:
       architecture: arm64
       channel: candidate
       version: "4.12"
+  source:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
```

Comment:
```suggestion
      version: "4.12"
```

Reply:
same to https://github.com/openshift/release/pull/57024/files#r1772434496